test-case/simultaneous-playback-capture.sh

```diff
@@ -54,8 +70,10 @@ do
     dlogi "Testing: (Loop: $i/$loop_cnt)"
     # clean up dmesg
     sudo dmesg -C
-    for idx in $(seq 0 $(expr $PIPELINE_COUNT - 1))
+    # follow sof-tplgreader the capture will after playback
```

Comment:
I'm not sure what you're trying to say here. Can you expand on this a bit?

Reply:
@stripes416 check with patch #80 the both will split 2 pipeline the order of those pipeline is 1st is playback, 2nd is capture, so here step change to 2

Reply:
Okay, I think I understand what you're trying to say here now. Maybe change the comment to something like below:

```suggestion
    # following sof-tplgreader, split 'both' pipelines into separate playback & capture pipelines, with playback occurring first
```